devops/kube/frontend_manifest.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata: 
+  name: forge-frontend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: forge-frontend
+  template:
+    metadata:
+      labels:
+        app: forge-frontend
+    spec:
+      containers:
+      - name: forge-frontend
+        image: dajpearl/forge-frontend:latest
```

Comment:
do we need the dajpearl prefixes to the repositories when it's our own ACR?

Reply:
Yes, since we are currently using Dockerhub for our images not ACR. We were waiting on terraform for the ACR since it needs  an azure subscription.